### PR TITLE
fix: Migration from tech-preview-channel

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

## What does this pull request change
Adapt migration script to reuse them for migration from tech-preview-channel

## What issues does this pull request fix or reference
https://issues.redhat.com/browse/CRW-2357

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by max-cx** [comment]
LGTM. @themr0c is better to review the Bash script changes (I've only started learning Bash recently).

File: modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc [hunk 1]
```diff
@@ -23,6 +23,7 @@ The workspace engine and authentication system used until {prod-prev-short} vers
 [source,bash,subs="+attributes"]
 ----
 export INSTALLATION_NAMESPACE={prod-namespace} #<1>
+export OPERATOR_NAMESPACE={prod-namespace}     #<1>
```
Comment (nickboldt) [inline modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc:26]: does this default to a different value than `INSTALLATION_NAMESPACE` ? Does it need a different footnote number?

Comment (tolusha) [reply modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc:26]: 1. Migration from `latest` channel: OPERATOR_NAMESPACE = INSTALLATION_NAMESPACE
2. Migration from`tech-preview-channel` OPERATOR_NAMESPACE != INSTALLATION_NAMESPACE  
So, we need to differentiate these 2 namespaces

**Comment by themr0c** [comment]
@tolusha Can you help set the milestone? 
* 7.46 if we need to backport to Dev Spaces 3.0
* else 7.47 or 7.48

**Comment by tolusha** [comment]
Done

**Pull request merged**